fadtk/model_loader.py

```diff
@@ -54,16 +56,21 @@ def _get_embedding(self, audio: np.ndarray):
     def load_wav(self, wav_file: Path):
         wav_data, _ = soundfile.read(wav_file, dtype='int16')
         wav_data = wav_data / 32768.0  # Convert to [-1.0, +1.0]
-
+        if self.audio_len is not None:
+            if wav_data.shape[0] < self.sr*4:
+                padding_size = self.sr*4 - len(wav_data)
+                wav_data = np.pad(wav_data, (0, padding_size), 'constant', constant_values=(0, 0))
```

Comment:
`wav_data.ndim` be either 1 or 2 depending on the channel number, as we didn't force `always_2d=True`. if stereo files were used, this code might not work.. right? so, i) let's add a comment explaining we're expecting mono file ii) check if it's actually true, and if not -- if the file is stereo -- let's `raise ValueError` here.